subworkflows/local/generate_downstream_samplesheets/main.nf

```diff
@@ -0,0 +1,112 @@
+//
+// Subworkflow with functionality specific to the nf-core/createtaxdb pipeline
+//
+
+workflow SAMPLESHEET_TAXPROFILER{
+    take:
+    ch_reads
+
+    main:
+    ch_header  = Channel.empty()
+    format     = 'csv' // most common format in nf-core
+    format_sep = ','
+
+    // Make your samplesheet channel construct here depending on your downstream
+    ch_list_for_samplesheet = ch_reads
+                                .map {
+                                    meta, reads ->
+                                        def out_path            = file(params.outdir).toString() + '/filter/filtered/'
```

Comment:
```suggestion
                                        def out_path            = file(params.outdir).toString() + '/filter/'
```

Where does the `filtered/` bit come from/ I don't see it in the output docs nor in `modules.conf`?